README.md

```diff
@@ -59,38 +54,82 @@ This will provide you with the packages needed to run the `test_discovery.py` sc
-
-1) Copy an example folder or make one with the same structure
 
-2) Create test cases in tests.txt with the following format:
+## Discovery Examples Directory Overview
+Each discovery example contains a folder named `custom`, which in turn contains the required two items for launching Discovery: `intents.json` and the `entities` folder.
+Some examples contain a `schemas.json` file to customize the return json.
+They also contain example scripts to see how the particular configuration will detect entities.
 
 ```
-test: {name of test}
-transcript: {transcript you want to parse}
-{entity 1}: {value}
-{entity 2}: {value}
-...
+examples
+└── room_dialing
+    ├── custom
+    │   ├── entities
+    │   │   └── digit.py
+    │   └── intents.json
+    ├── send_transcript_to_discovery.sh
+    └── tests.txt
 ```
 
-Ensure that your *transcripts are unformatted text with numbers spelled out*. Formatting will be taken care of by your entities, and the output from transcription engines will be unformatted.
 
-3) Create your `intents.json` file to specify examples that resemble your tests.
-
-4) Create any additional entities required by your intents.
-
-5) Edit the credentials and other configuration parameters in `discovery_config.py`
-
-6) Run `python3 test_discovery.py folder_name` to test your interpreter. For example:
-
-```
-python test_discovery.py examples/directions
-```
-
-### Using Compiled Binary Files
+## The 'room_dialing' Interpreter Example
+
+1) Test cases for the room dialing example are in `examples/room_dialing/tests.txt` 
+    ```
+    test: dial number
+    transcript: please dial eight
+    digit: 8
+
+    test: dial number
+    transcript: press one eight
+    digit: 1
+    ...
+    ```
+
+    `tests.txt` follows the following format:
+    ```
+    test: {name of test}
+    transcript: {transcript you want to parse}
+    {entity 1}: {value}
+    {entity 2}: {value}
+    ...
+    ```
+    Ensure that your *transcripts are unformatted text with numbers spelled out*. Formatting will be taken care of by your entities, and the output from transcription engines will be unformatted.
+
+2) The intents file, `examples/room_dialing/custom/intents.json`, contains examples that match the tests
+    ```
+    {
+      "intents": [
+        {
+          "label": "room_dialing",
+          "entities": ["digit"]
+        }
+      ]
+    }
+    ```
+    where "room_dialing" is the name of the intent and the entities value ("digit") match entities in `tests.txt`.
+
+3) Edit your `discovery_config.py` to specify your "GKT_USERNAME" and "GKT_SECRETKEY" credentials.
+
+4) Execute `python3 test_discovery.py examples/room_dialing` to test the room_dialing example.
+    ```
+    Discovery Ready
+    ======
+    Testing: dial number
+    Test passed
+    
+    ======
+    Testing: dial number
+    Test passed
+    
+    
+    (2 / 2) tests passed in 0s with 0 errors, Character error rate: 0.00%
+    ```
+
+## Using Compiled Binary Files
```

Comment:
I would weakly suggest putting this in the dependencies section and adding some boilerplate about how the test launches Discovery Docker container.

Reply:
Placing the test_discovery script in the dependencies section seems out of place. It is a test of an example folder.

I added information about how the test launches our Discovery Docker container and reference this in the dependencies section.

Reply:
After talking with Matthew, we decided to move the `Using Compiled Binary Files` section to be just after the `Dependencies` section